pkg/kubelet/pluginmanager/operationexecutor/operation_generator.go

```diff
@@ -185,7 +195,7 @@ func (og *operationGenerator) notifyPlugin(client registerapi.RegistrationClient
 	return nil
 }
 
-// Dial establishes the gRPC communication with the picked up plugin socket. https://godoc.org/google.golang.org/grpc#Dial
+// dial establishes the gRPC communication with the picked up plugin socket. https://godoc.org/google.golang.org/grpc#Dial
 func dial(unixSocketPath string, timeout time.Duration) (registerapi.RegistrationClient, *grpc.ClientConn, error) {
 	ctx, cancel := context.WithTimeout(context.Background(), timeout)
 	defer cancel()
```

Comment:
Does anything need to change in `pkg/kubelet/pluginmanager/metrics/metrics.go`?

Reply:
I don't think so